resolwe_bio/kb/models.py

```diff
@@ -101,20 +117,35 @@ class Mapping(models.Model):
     class Meta:
         """Mapping Meta options."""
 
-        unique_together = [
-            [
-                "source_db",
-                "source_id",
-                "source_species",
-                "target_db",
-                "target_id",
-                "target_species",
-                "relation_type",
-            ],
+        constraints = [
+            models.UniqueConstraint(
+                fields=[
+                    "source_db",
+                    "source_id",
+                    "source_species",
+                    "target_db",
+                    "target_id",
+                    "target_species",
+                    "relation_type",
+                ],
+                name="uniq_mapping_source_target_type",
+            ),
         ]
-        index_together = [
-            ["source_db", "source_id", "source_species", "target_db"],
-            ["target_db", "target_id", "target_species"],
+        indexes = [
+            models.Index(
+                name="idx_feature_source_target",
+                fields=[
+                    "source_db",
+                    "source_id",
+                    "source_species",
+                    "target_db",
+                    "target_species",
```

Comment:
Should target_species also be added to the first index?

Reply:
What do you mean by "first index"?